docs/azure-saas-docs/content/en/components/Identity/identity-framework-setup-manual.md

```diff
@@ -0,0 +1,392 @@
+  | tenant.delete | Delete a customer's own Tenant data |
+  | tenant.global.delete | Admin-level delete permissions for all Tenants |
+
+#### Module API Permissions
+- From the `App registration` view
+- For the apps listed below, navigate to them, then navigate to `API permissions` in their navbars
+- Add the following API permissions on the associated module:
+- asdk-saas-app
+  - tenant.read
+- asdk-signupadmin-app
+  - tenant.read
+  - tenant.global.read
+  - tenant.write
+  - tenant.global.write
+  - tenant.delete
+  - tenant.global.delete
+  - Delete tenant global
+
+### Step 5. Create self-signed certificate
+
+>Note: A self-signed certificate is not recommended for production environments. Read more [here](https://azure.github.io/azure-saas/components/identity/permissions-service/#authentication).
+
+- Create an RSA self-signed certificate to be referenced and uploaded to your Azure AD B2C Tenant
+- The .pfx file will be uploaded in a later step to your Azure AD B2C to allow it access to integrate with your Permissions API
+- OpenSSL with [Powershell Core (v7.0+)](https://github.com/PowerShell/PowerShell) Example:
+
+```
+$CertificatePassword = Read-Host -AsSecureString
+
+openssl req -newkey rsa:4096 -x509 -sha256 -days 365 -nodes -out certificate.crt -keyout certificate.key -subj         "/CN=*.azurewebsites.net"
+
+$pswd = ConvertFrom-SecureString -SecureString $CertificatePassword
+
+openssl pkcs12 -export -out selfSignedCertificate.pfx -inkey certificate.key -in certificate.crt -password pass:$pswd
+```
+
+
+### Step 6. Create Policy Keys in Azure AD B2C Tenant
+
+![(Identity Experience Framework Screenshot)](/azure-saas/images/identity-framework-manual-step-6-identity-experience-framework.png)
+
+- Find `Policy keys` in `Azure AD B2C Service` by navigating to `Identity Experience Framework`
+- Click Add and select the following options to generate the system policy keys
+
+![(Policy Keys Screenshot)](/azure-saas/images/identity-framework-manual-step-6-policy-keys.png)
+
+- TokenSigningKeyContainer
+  - Options: Generate
+  - Name: TokenSigningKeyContainer
+  - Key type: RSA
+  - Key usage: Signature
+- TokenEncryptionKeyContainer
+  - Options: Generate
+  - Name: TokenEncryptionKeyContainer
+  - Key type: RSA
+  - Key usage: Encryption
+- RestApiClientCertificate
+  - Options: Upload
+  - Name: *.azurewebsites.net
+  - File upload: (The .pfx created in step 5)
+  - Password: (The password created in step 5)
+
+### Step 7. Clone ASDK repo
+
+- Clone the latest release of the [ASDK](https://github.com/Azure/azure-saas)
+
+### Step 8. Initiate identity Bicep deployment with parameters from previous few steps
+
+- Generate a standard [Bicep parameters file](https://docs.microsoft.com/en-us/azure/azure-resource-manager/bicep/parameter-files) 
+- Navigate to `Saas.identity/Saas.Identity.Iac` for the `az deploy` to work correctly
+- Deploy identity Bicep with the following parameters
```

Comment:
Add the command here that they can copy/paste. You can find it on line 552 of the b2c-create script.